test/test_contrib.py

```diff
@@ -36,6 +36,66 @@ def test_normalize(self, device, dtype):
         assert_close(out.max(), expected_max)
 
 
+class TestKMeans:
```

Comment:
from kornia.testint import BaseTester

```suggestion
class TestKMeans(BaseTester):
```